apps/services/regulations-admin-backend/src/app/modules/draft_regulation/draft_regulation.model.ts

```diff
@@ -83,7 +83,7 @@ export class DraftRegulationModel extends Model<DraftRegulationModel> {
     type: DataType.STRING,
   })
   @ApiProperty()
-  ministry_id?: MinistrySlug
+  ministry?: string
```

Comment:
Might want to remove the `MinistrySlug` import above

Reply:
done